frontend/src/metabase/visualizations/visualizations/LinkViz/LinkViz.unit.spec.tsx

```diff
@@ -201,6 +202,46 @@ describe("LinkViz", () => {
   });
 
   describe("entity links", () => {
+    it("wraps the dashboard's description", async () => {
+      const longDescription =
+        "This is a very very very very very very very very very very long dashboard description. Boy, I sure hope it does not overflow the screen when it's displayed in a tooltip. The only thing that would make it worse is if the text didn't wrap.";
+
+      const card = createMockLinkDashboardCard({
+        visualization_settings: {
+          link: {
+            entity: {
+              id: 1,
+              db_id: 20,
+              name: "Table Une",
+              model: "table",
+              description: longDescription,
+            },
+          },
+        },
+      });
+
+      renderWithProviders(
+        <LinkViz
+          dashboard={createMockDashboard({})}
+          dashcard={card}
+          isEditing={false}
+          settings={card.visualization_settings as LinkCardVizSettings}
+          onUpdateVisualizationSettings={jest.fn()}
+        />,
+      );
```

Comment:
As you've probably seen, our test files usually have a `setup` function that creates the component, and that function calls `render` or `renderWithProviders`. Would it be useful to re-use the setup function here?